### PR TITLE
Fix accessibility for tab panels in general and inspector tabs in particular

### LINKED ISSUE #933: Refactor Inspector tabs for accessibility
## Feature Description

At some point the library (left-hand side) was refactored for better keyboard accessibility.

The same should be done for the inspector (right-hand side) as well.

Lots of potential to abstract this to make it more DRY.

Tests needed to verify intended behavior.

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by barklund**
## Summary

This makes sure that tab lists, tabs and tab panels all follow [the specification](https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel).

## Relevant Technical Choices

* Add label for tablist
* <kbd>Home</kbd> and <kbd>End</kbd> can be used to move to first/last tab respectively.
* Inspector tab panels are always in DOM, but only current is not `hidden`.

## User-facing changes

N/A

## Testing Instructions

N/A

---

Fixes #933

### CODE REVIEW

**Review by miina — COMMENTED** [review]
Generally looks good and works great.

Two notes:
1. There are some console errors, some of these seem to be on the main branch as well (but not all), one example:
Screenshot 2020-08-12 at 10 14 29

2. Do any tests need updating with the new labels etc., too?

File: assets/src/edit-story/components/inspector/inspectorLayout.js [hunk 1]
```diff
@@ -60,6 +60,7 @@ function InspectorLayout() {
   return (
     <Layout ref={inspector}>
       <TabView
+        label={__('Inspector selection', 'web-stories')}
```
Comment (swissspidy) [inline assets/src/edit-story/components/inspector/inspectorLayout.js]:
```suggestion
        label={__('Inspector Selection', 'web-stories')}
```

For consistency with `Element Library Selection`

File: assets/src/edit-story/components/tabview/index.js [hunk 2]
```diff
@@ -93,48 +92,64 @@ function TabView({
...
   );
 
-  const handleNavigation = useCallback(
-    (direction) => {
-      const currentIndex = tabs.findIndex(({ id }) => id === tab);
-      const nextTab = tabs[currentIndex + direction];
+  const selectTabByIndex = useCallback(
+    (index) => {
+      // Index wraps, so -1 is last element
+      const nextTab = tabs[index < 0 ? tabs.length - index : index];
```
Comment (merapi) [inline assets/src/edit-story/components/tabview/index.js]:
```suggestion
      const nextTab = tabs[index < 0 ? tabs.length + index : index];
```

**Review by merapi — APPROVED** [review]
A minor fix and it's great.

**Comment by barklund** [comment]
> Generally looks good and works great.
> 
> Two notes:
> 
> 1. There are some console errors, some of these seem to be on the main branch as well (but not all), one example:

I'll double check if it's related to this PR. If not, I'll try to fix in a separate one.

> 2. Do any tests need updating with the new labels etc., too?

All tests pass just fine. I guess it's partially because those labels aren't really necessary (the label for the list of tabs and the label for the tab panel itself), so we don't use those directly for selection, nor do we have any tests for the inactive tab panels (for good reason).

**Comment by barklund** [comment]
> I'll double check if it's related to this PR. If not, I'll try to fix in a separate one.

@miina, it was due to the document panel being in the DOM (but `hidden`) before the story was fully loaded. I added defaults in e5cb00a, and all errors are now suppressed.

**Review by miina — APPROVED** [review]
LGTM

**Pull request merged**